.github/workflows/build.yml

```diff
@@ -0,0 +1,150 @@
+name: build
+
+# We now default to running this workflow on every push to every branch.
+# This provides fast feedback when build issues occur, so they can be
+# fixed prior to being merged to the main branch.
+#
+# If you want to opt out of this, and only run the build on certain branches
+# please refer to the documentation on branch filtering here:
+#
+#   https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore
+#
+#on: [workflow_dispatch, push]
```

Comment:
We'll uncomment this sooner or later. If you'd like, uncomment it now & save one PR :)

(and remove the other `on` bit)

Reply:
Ah, good catch, I'll fix that now 👍🏻

Reply:
c98fb2b2e46da93a8b32971df019e9a423be001a ⚡